locidex/example/format_db_mlst_out/results.json

```diff
@@ -1,8 +1,8 @@
 {
-    "analysis_start_time": "2024-02-26 16:07:42",
+    "analysis_start_time": "2024-04-04 15:51:06",
     "parameters": {
-        "input": "/Users/jrobertson/PycharmProjects/locidex/locidex/example/format_db_mlst_in",
-        "outdir": "/Users/jrobertson/PycharmProjects/locidex/locidex/example/format_db_mlst_out",
+        "input": "locidex/example/format_db_mlst_in",
+        "outdir": "/tmp/pytest-of-mwells/pytest-115/build0",
```

Comment:
I was wondering if you can use a global variables for paths instead of absolute paths for better portability such as `TEST_ROOT = os.path.dirname(__file__)`. Take a look at https://github.com/phac-nml/mob-suite/blob/master/mob_suite/tests/test_mobtyper_vs_biomarker_report.py